src/core/server/http/http_tools.ts

```diff
@@ -70,6 +70,7 @@ export function getServerOptions(config: HttpConfig, { configureTLS = true } = {
       key: readFileSync(ssl.key!),
       passphrase: ssl.keyPassphrase,
       secureOptions: ssl.getSecureOptions(),
+      requestCert: config.ssl.requestCert,
```

Comment:
It looks like on L58 that `ssl` is already mapped to `config.ssl`, so this can be shortened.